component/src/test/java/org/wso2/extension/siddhi/map/text/sinkmapper/TextCustomSinkMapperTestCase.java

```diff
@@ -838,9 +839,81 @@ public void fileSinkTestGroup() throws InterruptedException {
         }
     }
 
-    @BeforeMethod
-    void initVariable() {
-        wso2Count.set(0);
-        ibmCount.set(0);
+    /**
+     * Creating test for publishing events with TEXT mapping.
+     *
+     * @throws Exception Interrupted exception
+     */
+    @Test
+    public void testHTTPTextMappingText() throws Exception {
+
+        log.info("Creating test for publishing events with TEXT mapping.");
+        SiddhiManager siddhiManager = new SiddhiManager();
+        siddhiManager.setExtension("text-output-mapper", TextSinkMapper.class);
+        String inStreamDefinition = "Define stream FooStream (message String,method String,headers String);"
+                + "@sink(type='http',publisher.url='http://localhost:8005/abc',method='{{method}}',"
+                + "headers='{{headers}}',"
+                + "@map(type='text', @payload('{{message}}'))) "
+                + "Define stream BarStream (message String,method String,headers String);";
+        String query = ("@info(name = 'query1') " +
+                "from FooStream select message,method,headers insert into BarStream;");
+        SiddhiAppRuntime siddhiAppRuntime = siddhiManager.createSiddhiAppRuntime(inStreamDefinition +
+                query);
+        InputHandler fooStream = siddhiAppRuntime.getInputHandler("FooStream");
+        siddhiAppRuntime.start();
+        HttpServerListenerHandler lst = new HttpServerListenerHandler(8005);
```

Comment:
Please use more meaningful names for variables, other than 'lst' :)

Reply:
Fixed in cdb7b86019f370bcb90c545043b2ad205e8e98df